### PR TITLE
Set routes current and common to 2.0 by default

### PR DESCRIPTION
**Pull request by keedya**
@amymullins @heckj @RackHD/corecommitters @dalebremner @srinia6 @brianparry

### CODE REVIEW

**Comment by keedya** [comment]
@cgx027, I had to modify unit test to make sure it works regardless to the api version
Please take a look at this PR to make sure I have it all covered.

**Comment by dalebremner** [comment]
+1

File: lib/services/http-service.js [hunk 1]
```diff
@@ -220,7 +220,7 @@ function httpServiceFactory(
         );
 
         //re-route common and current
-        var versionPath = configuration.get('versionBase') || '1.1';
+        var versionPath = configuration.get('versionBase') || '2.0';
```
Comment (benbp) [inline lib/services/http-service.js]: I recommend switching to use the standard way we do configurations defaults (which obviously happened after the original code here was written):

`configuration.get('versionBase', '2.0')`

**Comment by benbp** [comment]
👍

File: spec/lib/services/notification-api-service-spec.js [hunk 2]
```diff
@@ -36,86 +37,42 @@ describe('Http.Api.Notification', function () {
...
-            .post(
-                '/api/current/notification?taskId='
-                + notificationMessage.taskId 
-                + '&data='
-                + notificationMessage.data)
-            .set('Content-Type', 'application/json')
-            .expect('Content-Type', /^application\/json/)
-            .expect(201, notificationMessage);
+            return notificationApiService.postNotification(notificationMessage)
+                        .then(function (resp) {
```
Comment (yyscamper) [inline spec/lib/services/notification-api-service-spec.js]: indent problem

File: spec/lib/services/notification-api-service-spec.js [hunk 3]
```diff
@@ -36,86 +37,42 @@ describe('Http.Api.Notification', function () {
...
+                            console.log(resp);
```
Comment (yyscamper) [inline spec/lib/services/notification-api-service-spec.js]: Avoid log some useless info

**Comment by yyscamper** [comment]
@keedya : Please revert all file permissions from 755 to 644

File: lib/services/http-service.js (hunk 1 above)
Comment (yyscamper) [inline lib/services/http-service.js]: Also my recommendation, this is the common way we handle the default value before.

**Comment by cgx027** [comment]
Hi @keedya , the modification looks good to me. There are background talks between @benbp @heckj @yyscamper and myself on refactoring the notification api, so that I will need to push another PR to change the design in coming days. I do not see a reason to block this PR anyway.

**Comment by yyscamper** [comment]
:+1: with some minor suggestion.

**Comment by anhou** [comment]
:+1:  with comments fix

**Pull request merged**